src/examples/elevator-call-demo.ts

```diff
@@ -42,21 +42,15 @@ const onWebSocketMessage = (data: string): void => {
 const start = async () => {
   checkRequiredVariables()
 
-  // Fetch the first token which will by default contain application/inventory scope for our use in the next request
-  let accessToken = await fetchAccessToken(CLIENT_ID, CLIENT_SECRET)
+  // Fetch the access token with both application/inventory scope and access to do lift calls to
```

Comment:
Proposals to fix some style/typo issues for this to be more aligned with copy at the API documentation:

Fetch the access token with both application/inventory scope and access **to execute elevator calls on any building accessible** to the application - **note** that if you have many (100+) **resources, you** cannot use wildcards